cmd/vic-machine/create/create.go

```diff
@@ -1389,6 +1417,8 @@ func (c *Create) Run(clic *cli.Context) (err error) {
 
 	log.Infof("Initialization of appliance successful")
 
+	// We must check for the volume stores that are present after the portlayer presents.
```

Comment:
ah yes that was supposed to have a TODO in it so that I could search for it... thanks :D